### ISSUE TITLE
Current typescript ruleset requires eslint-plugin-typescript as a "peer" dependency

### ISSUE THREAD

**Issue by behalka** [strvcom/eslint-config-javascript#47]
Steps to reproduce:

- I installed ruleset with this `.eslintrc.js` configuration:
```
'use strict'

module.exports = {
  parser: '@typescript-eslint/parser',
  plugins: ['@typescript-eslint'],

  extends: [
    '@strv/javascript/environments/nodejs/v10',
    '@strv/javascript/environments/nodejs/optional',
    '@strv/javascript/environments/typescript/recommended',
    '@strv/javascript/coding-styles/recommended',
  ],
}
```
- I got this error: 
```
Failed to load plugin typescript: Cannot find module 'eslint-plugin-typescript' ...
```
- If I comment out the ruleset with typescript, eslint works :)

The `@typescript-eslint` library went under some changes and the `eslint-plugin-typescript` was required before so I suppose it's only needed to bump some dependencies in our library :) Hope this helps.

Current workaround now is to install `eslint-plugin-typescript` as a dev dependency into my project and everything works :)

**Comment by robertrossmann** [comment]
This might be fixed in `9.2.0`, via #45 which also added the missing rule plugin (sorry for that, my bad 😇), could you try once more with the new version and let me know? Thanks!

**Comment by behalka** [comment]
@robertrossmann I can confirm it's fixed :) 🎉
